src/BoE.xcodeproj/project.pbxproj

```diff
@@ -2279,22 +2285,17 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				COPY_PHASE_STRIP = YES;
-				DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
```

Comment:
Ahhh, okay. I recall doing a lot of pushing settings up to the project level instead of target level awhile back, not sure if I pushed it though (I have several commits that I haven't pushed yet because they'll break MSVC).